include/u.h

```diff
@@ -38,10 +38,8 @@ extern "C" {
 #		define __MAKECONTEXT_V2_SOURCE 1
 #	endif
 #endif
-// #define _BSD_SOURCE 1
```

Comment:
ah, i see you took these out, but that was part of this commit.  it should be part of the previous commit.  that way, the previous commit makes sense.